README.md

```diff
@@ -46,6 +47,10 @@ var cjsTree = typescript(inputTree, {
 
 An optional string, which when provide should be a descriptive annotation. Useful for debugging, to tell multiple instances of the same plugin apart.
 
+`throwOnError`
+
+An optional boolean, defaulting to true. If set to true, will cause the build to break on errors.
```

Comment:
I think it's actually `false` by default.

Reply:
oops yup